src/environ/_environ_config.py

```diff
@@ -63,6 +90,8 @@ def config(
        *generate_help*
     .. versionadded:: 20.1.0
        *frozen*
+    .. versionchanged:: 21.1.0
+       *prefix* now defaults to *PREFIX_NOT_SET* instead of `None`.
```

Comment:
Uh prefix wasn't None, it was "APP", no? I don't think this needs a change entry unless you can think of reasons this being not backward-compatible.

Reply:
Ah yes, correct. My bad! Sorry for the delay on this also. I'll amend this over the weekend

Reply:
Hi, @hynek. So sorry for the delay on this! I've amended this to read "APP" instead, and I've also amended the keyword arg documentation for `prefix=` above to make it more clear what the current behaviour is.